weave/trace_server/calls_query_builder.py

```diff
@@ -781,7 +809,12 @@ def process_operand(operand: "tsi_query.Operand") -> str:
             )
         elif isinstance(operand, tsi_query.GetFieldOperator):
             structured_field = get_field_by_name(operand.get_field_)
-            field = structured_field.as_sql(param_builder, table_alias)
+            if not use_agg_fn and isinstance(structured_field, CallsMergedAggField):
+                structured_field.use_agg_fn = False
+                field = structured_field.as_sql(param_builder, table_alias)
```

Comment:
i would discourage this pattern. Prefer to have `use_agg_fn` a parameter of as_sql, rather than flipping a property

Reply:
Hey! Coderrabbit had a good comment below! Exactly correct